src/browser/tab/mod.rs

```diff
@@ -718,6 +718,17 @@ impl Tab {
             .root)
     }
 
+    /// Get the full HTML contents of the page.
+    /// 
+    /// Equivalent to the following JS: ```document.documentElement.outerHTML```
+    pub fn get_content(&self) -> Result<String> {
+        let html = self
+            .evaluate("(function () { return document.documentElement.outerHTML })();", false)?
+                .value
+                .unwrap();
+        Ok(String::from(html.as_str().unwrap()))
+    }
```

Comment:
same with this https://chromedevtools.github.io/devtools-protocol/tot/DOM/#method-getOuterHTML

Reply:
refactor the code to this ref https://github.com/puppeteer/puppeteer/blob/dd275590903368aab107912752b975f3b0a16489/src/common/DOMWorld.ts#L253-L264